docs/introduction.rst

```diff
@@ -11,106 +11,143 @@ or modifying files produced by other programs (e.g.,
-its genealogy, and many operations on tree sequences act on the sample nodes
-by default (see the
-`tskit data model <https://tskit.readthedocs.io/en/latest/data-model.html#nodes-genomes-or-individuals>`_
-for more on this topic).
-The easiest way to pull out individuals alive at a particular time
-is with the :meth:`.SlimTreeSequence.individuals_alive_at()` method:
-see :ref:`sec_extracting_individuals` for more discussion.
-
-Here's a reminder of the terminology:
-
+In the recorded tree sequence, the alive individuals have all had their nodes marked as
+*sample nodes* which, as you may recall, means we have full ancestral information about
+them. As shown in the diagram this means the sample nodes, and the individuals
+containing them, are always present in the tree sequence (it's also possible to
+mark additional nodes as samples: see :ref:`sec_remembering_individuals`)
+
+In contrast to the individuals containing sample nodes, you can see that all the other
+circles, representing historical (i.e. dead) individuals, have gone, although sometimes
+their nodes remain. By default, only individuals with sample nodes are recorded in the
+tree sequence; that means ancestral nodes lose any information about which individuals
+they were in (see :ref:`sec_retaining_individuals` if you need to retain this information).
+
+As well as the actual individuals, many historical *nodes*  have been removed too,
+along with the lines linking them to the rest of the genealogy (in tree-sequence-speak the
+lines are known as "edges"). The deleted nodes are simply those that are not needed to
+reconstruct the relationships between the sample nodes. For example we remove nodes
+leading to a dead end (e.g. in individuals who had no offspring). Similarly, as time goes
+on, successive recombination events can gradually reduce the genetic contribution of
+older nodes to the current generation; any historical nodes whose genetic contribution to
+the samples has been whittled down to zero are also removed. Finally, note that to
+reconstruct relationships between samples, we strictly only need to keep a node if it
+represents the genetic *most recent common ancestor* (MRCA) of at least two samples. So
+by default, we also remove historical nodes that are only "on the way to" a sample, but
+do not represent a split in the tree (to alter this behaviour, check out the
+``retainCoalescentOnly`` parameter of ``treeSequenceInitialize()``)
+
+*****************************************************
+What if I'm interested in historical individuals too?
+*****************************************************
+
+As we've seen, the tree sequence retains enough of the nodes (genomes) in the past to
+reconstruct the genetic ancestry of the currently alive individuals. But you might want
+more than that. For example, there may be individuals who are not alive any more, but
+whose complete ancestry you would like to know.  Or it may be that you want to access the
```

Comment:
", or you'd like to know how the final generation relates to the individuals alive at some previous time."